.gitignore

```diff
@@ -1,3 +1,7 @@
 *.pyc
 */__pycache__
-psqldump
\ No newline at end of file
+psqldump
+musicbrainz/.DS_Store
+/musicbrainz/data
```

Comment:
This directory is already in the repo...are you sure you want to ignore it?

Reply:
> Since I'm working on another branch at the same time, how can I avoid having to choose the files to commit each time without adding these to the .gitignore?

I'm not sure exactly what the problem you are running into, but a temporary commit or `git stash` might do it for you. Choosing which files to include in a commit feels like very common practice to me, and certainly better than adding something to gitignore unless you actually intend to ignore the files...